tools/crm_resource_runtime.c

```diff
@@ -1331,7 +1331,7 @@ update_scheduler_input(pcmk_scheduler_t *scheduler, xmlNode **xml)
         scheduler->input = *xml;
         scheduler->priv->now = crm_time_new(NULL);
     }
-    return pcmk_rc_ok;
+    return rc;
```

Comment:
We'll need this one for 2.1.9-rc2, do a separate PR for this (listing as regression in 2.1.8) then backport it

Reply:
Noted. This wasn't even supposed to be in this branch. I had it as a precursor to trying to fix another crm_resource bug.